### PR TITLE
Institute page

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

**Comment by northwestwitch** [comment]
Do you need a review?

**Comment by moonso** [comment]
Sure, I just wanted to get a skeleton in place for future stuff

File: scout/server/app.py [hunk 1]
```diff
@@ -100,6 +101,7 @@ def register_blueprints(app):
     app.register_blueprint(api.api_bp)
     app.register_blueprint(alignviewers.alignviewers_bp)
     app.register_blueprint(phenotypes.hpo_bp)
+    app.register_blueprint(institutes.institutesov_bp)
```
Comment (northwestwitch) [inline scout/server/app.py]: Don't know how final the name of this page is, but I don't like much 'institutesov'. Stands for institute overview right? what about just overview?

File: scout/server/blueprints/institutes/views.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        for user_mail in ins_obj.get('sanger_recipients',[]):
+            user_obj = store.user(user_mail)
+            if not user_obj:
+                continue
+            sanger_recipients.append(user_obj['name'])
+        institutes.append(
+            {
+                'display_name': ins_obj['display_name'],
+                'internal_id': ins_obj['_id'],
+                'coverage_cutoff': ins_obj['coverage_cutoff'],
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/views.py]: This is causing a key error while testing with the demo data:
'coverage_cutoff': ins_obj['coverage_cutoff'],
This is causing a key error if you test with the demo data:
``` 
'coverage_cutoff': ins_obj['coverage_cutoff'],
KeyError: 'coverage_cutoff'
``` 
We should either fix the demo institute to have a cutoff key or this code to use ins_obj.get('coverage_cutoff') instead

Comment (moonso) [reply scout/server/blueprints/institutes/views.py]: 👍 I'll fix that

**Comment by moonso** [comment]
Ok I have fixed according to your suggestions. Can you approve @northwestwitch ?

**Comment by northwestwitch** [comment]
One last thing. You named the html template in institutes/templates/overview 'index.html'. I know it's just a name but I think that to be consistent with the rest of the app it should be renamed to 'institutes.html'.

File: scout/server/blueprints/institutes/views.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import logging
+
+from pprint import pprint as pp
+
+from flask import (abort, Blueprint, current_app, redirect, render_template,
+                   request, url_for, send_from_directory, jsonify, Response, flash)
+from flask_login import current_user
+
+from dateutil.parser import parse as parse_date
+from scout.constants import PHENOTYPE_GROUPS
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/views.py]: You don't need most of these imports right now

Comment (northwestwitch) [reply scout/server/blueprints/institutes/views.py]: Not PHENOTYPE_GROUPS, but those flask ones above

**Pull request merged**